pkg/collector/python/test_loader.go

```diff
@@ -139,7 +140,11 @@ func testLoadCustomCheck(t *testing.T) {
 	}
 
 	rtloader = newMockRtLoaderPtr()
-	defer func() { rtloader = nil }()
+	pythonOnce.Do(func() {})
```

Comment:
Could we use `mockRtloader` utility test function on this tests as well?

Reply:
Yes good idea ! Done